modules/util/string.go

```diff
@@ -95,3 +95,12 @@ func UnsafeBytesToString(b []byte) string {
 func UnsafeStringToBytes(s string) []byte {
 	return unsafe.Slice(unsafe.StringData(s), len(s))
 }
+
+// GetFirstCharacters returns the first n characters of a string
+func GetFirstCharacters(s string, n int) string {
```

Comment:
Is `EllipsisString` better?

Reply:
Ah, yes. Just use `base.TruncateString` is OK. The code is duplicated.